molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/resources/Resources.java

```diff
@@ -24,4 +27,11 @@
 	 */
 	Iterable<Entity> findAll(String name, Query q);
 
+	/**
```

Comment:
To be clear, this is about the "getResourcesNames()" not the "findAll" that one we obviously need.
Github shows it a bit confusing because I create the line note on the first line of the comment.